identity/app/implicits/Articles.scala

```diff
@@ -11,6 +12,32 @@ trait Articles {
     lazy val href = "%s/%s".format(conf.Configuration.site.host, savedArticle.id)
     lazy val savedAt = fmt.print(savedArticle.date)
   }
+
+  implicit class RichSavedArticles(savedArticles: SavedArticles) {
+    val fmt = ISODateTimeFormat.dateTimeNoMillis()
+
+
+    def newestFirst = savedArticles.articles.reverse
+    def contains(shortUrl: String) : Boolean = savedArticles.articles.exists( sa => sa.shortUrl == shortUrl)
+
+    def addArticle(id: String, shortUrl: String): SavedArticles = {
+      var date = new DateTime()
```

Comment:
@robertberry because I was switching between scala and javascript and wasn't paying attention. I'll fix it.